statsmodels/tsa/statespace/kalman_filter.py

```diff
@@ -1112,11 +1112,11 @@ def predict(self, start=None, end=None, dynamic=None, full_results=False,
             if dynamic < 0:
                 raise ValueError('Dynamic prediction cannot begin prior to the'
                                  ' first observation in the sample.')
-            if dynamic > end:
+            elif dynamic > end:
```

Comment:
Ok, I have no experience for this and only recently figured out what `dynamic` even is supposed to be (in ARIMA).